packages/js/src/core/fetch-token.ts

```diff
@@ -49,5 +49,5 @@ export const fetchTokenByRefreshToken = async (
 
   const scopeValues = tokenSnakeCaseResponse.scope?.split(' ');
 
-  return camelcaseKeys({ ...tokenSnakeCaseResponse, scope: scopeValues });
+  return camelcaseKeys({ ...tokenSnakeCaseResponse, scopes: scopeValues });
```

Comment:
i think there's no need to normalize the response here. this should be done be the function caller.

Reply:
But as we discussed several days ago, the response type of this function has the prop called `scopes: string[]`, instead of `scope: string` any more.
So I think this is the correct behavior out of our discussion result.

Reply:
Agree with @gao-sun

Reply:
OK, aligned